packages/core/lib/commands/debug/loadConfig.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (options) {
+  const Config = require("@truffle/config");
+
+  let config;
+  try {
+    config = Config.detect(options);
+  } catch (error) {
+    if (options.url) {
```

Comment:
This is maybe being nitpicky, I think it would be good here to also check that the error is of the correct type (like, `error instanceof TruffleError`... maybe check the message too if you want to be really sure, although maybe that's overkill) before invoking this handling, and rethrowing otherwise.

Reply:
I wouldn't add the check for the message. If the message is changed in the `Config`, we have a non-functional code here

Reply:
added the `instanceof` check, but as I said, I don't think a check for the message is a good idea. Other 2 options would be: 
1. having a constant with the message in Config file - which doesnt look like a good idea
2. Having a more specific error `TruffleConfigFileNotFoundError`, which doesn't seem to be the direction the rest of the codebase is going

Anyhow, seems fine now

Reply:
added the `instanceof` check, but as I said, I don't think a check for the message is a good idea. Other 2 options would be: 
1. having a constant with the message in Config file - which doesnt look like a good idea
2. Having a more specific error `TruffleConfigFileNotFoundError`, which doesn't seem to be the direction the rest of the codebase is going

Anyhow, seems fine now

Reply:
Yeah, that makes sense.  Thanks!